### ISSUE TITLE
"Unapproved Report" Details Element Affordance Unclear

### ISSUE THREAD

**Issue by jkva** [w3c/aria-at-app#391]
The "Unapproved Report" details elements within test reports are styled in such a way that they do not visually suggest that they are interactive and/or can expand. Besides this, they look similar to the "Raise an Issue" anchor control, suggesting that they are perhaps anchors or buttons. Neither does the lack of a verb suggest user action.

![An animated partial screen capture of a "Unapproved Report" details element being expanded and collapsed.](https://user-images.githubusercontent.com/217264/145798947-159155a1-7c45-40f4-a0bd-76da5924f702.gif)
